src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/experimental/FlatFW.java

```diff
@@ -0,0 +1,249 @@
+ // TODO: license
+package org.reaktivity.nukleus.maven.plugin.internal.generated.experimental;
+
+import static java.lang.String.format;
+
+import java.nio.ByteOrder;
+import java.nio.charset.StandardCharsets;
+import java.util.BitSet;
+
+import javax.annotation.Generated;
+import org.agrona.BitUtil;
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+
+@Generated("reaktivity")
+public final class FlatFW extends Flyweight {
+  public static final int FIELD_OFFSET_FIXED1 = 0;
+
+  private static final int FIELD_SIZE_FIXED1 = BitUtil.SIZE_OF_BYTE;
+
+  public static final int FIELD_OFFSET_FIXED2 = FIELD_OFFSET_FIXED1 + FIELD_SIZE_FIXED1;
+
+  private static final int FIELD_SIZE_FIXED2 = BitUtil.SIZE_OF_SHORT;
+
+  public static final int FIELD_OFFSET_STRING1 = FIELD_OFFSET_FIXED2 + FIELD_SIZE_FIXED2;
+
+  public static final int FIELD_OFFSET_FIXED3 = 0;
+
+  private static final int FIELD_SIZE_FIXED3 = BitUtil.SIZE_OF_INT;
+
+  public static final int FIELD_OFFSET_STRING2 = FIELD_OFFSET_FIXED3 + FIELD_SIZE_FIXED3;
+
+  private final StringFW string1RO = new StringFW();
+
+  private final StringFW string2RO = new StringFW();
+
+  public int fixed1() {
+    return buffer().getByte(offset() + FIELD_OFFSET_FIXED1) & 0xFF;
+  }
+
+  public int fixed2() {
+    return buffer().getShort(offset() + FIELD_OFFSET_FIXED2, ByteOrder.BIG_ENDIAN) & 0xFFFF;
+  }
+
+  public StringFW string1() {
+    return string1RO;
+  }
+
+  public int fixed3() {
+    return buffer().getInt(string1().limit() + FIELD_OFFSET_FIXED3);
+  }
+
+  public StringFW string2() {
+    return string2RO;
+  }
+
+  @Override
+  public FlatFW wrap(DirectBuffer buffer, int offset, int maxLimit) {
+    super.wrap(buffer, offset, maxLimit);
+    string1RO.wrap(buffer, offset + FIELD_OFFSET_STRING1, maxLimit);
+    string2RO.wrap(buffer, string1RO.limit() + FIELD_OFFSET_STRING2, maxLimit);
+    checkLimit(limit(), maxLimit);
+    return this;
+  }
+
+  @Override
+  public int limit() {
+    return string2().limit();
+  }
+
+  @Override
+  public String toString() {
+    return String.format("FLAT [fixed1=%d, fixed2=%d, string1=%s, fixed3=%d, string2=%s]", fixed1(), fixed2(), string1RO.asString(), fixed3(), string2RO.asString());
+  }
+
+  public static final class Builder extends Flyweight.Builder<FlatFW> {
+    private static final int FIELD_COUNT = 5;
+    private static final int DEFAULT_FIXED2 = 222;
+    private static final int DEFAULT_FIXED3 = 333;
+
+    private static final int INDEX_FIXED1 = 0;
+    private static final int INDEX_FIXED2 = 1;
+    private static final int INDEX_STRING1 = 2;
+    private static final int INDEX_FIXED3 = 3;
+    private static final int INDEX_STRING2 = 4;
+
+    private static final BitSet FIELDS_WITH_DEFAULTS = new BitSet(FIELD_COUNT);
```

Comment:
`BitSet` is needed for more than `64` fields, otherwise `long` is sufficient here.

Reply:
Noted, I will file a bug when the changes get merged

Reply:
thanks